### PR TITLE
Rationalize and clarify IANA Considerations

### PR DESCRIPTION
**Pull request by TallTed**
- Tweak subsection titles of `Media Type Registrations`
- Make `JWT Claim Registrations` a parallel subsection

***
<a href="https://pr-preview.s3.amazonaws.com/TallTed/vc-data-model/pull/1163.html" title="Last updated on Jul 12, 2023, 1:42 AM UTC (46743e8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1163/85724de...TallTed:46743e8.html" title="Last updated on Jul 12, 2023, 1:42 AM UTC (46743e8)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Review by msporny — APPROVED** [review]

File: index.html
Comment (TallTed) [inline index.html:5083]: To match the other two retitled sections...
```suggestion
  <h2>JSON Web Token Claim Registrations: `vc` and `vp`</h2>
```

**Comment by TallTed** [comment]
It *might* make sense to split the `JWT Claim Registrations` into one each for `vc` and `vp`, but these are enough simpler than the `Media Type Registrations` that this may not be necessary. I will leave that to the editors to decide.

File: index.html
Comment (OR13) [inline index.html:5112]:
```suggestion
  <h2>application/vc+ld+json</h2>
```

File: index.html
Comment (OR13) [inline index.html:5167]:
```suggestion
  <h2>application/vp+ld+json</h2>
```

**Review by OR13 — CHANGES_REQUESTED** [review]
@TallTed I suggested changes to align with the way the JSON-LD WG has requested registrations, see also https://github.com/w3c/json-ld-syntax/pull/415

I suggest you make your suggestions on that open PR, and if accepted we merge this PR with your changes, if that working group does not accept your changes, I don't think we should deviate from how they are registering things.

File: index.html
Comment (OR13) [reply index.html:5112]: https://github.com/w3c/json-ld-syntax/pull/415/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R13429

File: index.html
Comment (OR13) [reply index.html:5167]: https://github.com/w3c/json-ld-syntax/pull/415/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R13429

File: index.html
Comment (TallTed) [reply index.html:5112]: @OR13 — 
1. Why delete `Media Type Registration:`?
2. What is the relevant piece of that GIANT diff to which you linked?

Saving yourself writing time, as here, radically increases the time your readers must devote to deciphering your meaning. Please don't do this.

**Comment by TallTed** [comment]
> [@OR13] @TallTed I suggested changes to align with the way the JSON-LD WG has requested registrations, see also https://github.com/w3c/json-ld-syntax/pull/415

That appears to be your own PR, and the "alignment" you seem to be suggesting here appears to be with that PR (i.e., with yourself), not with the JSON-LD WG.

> [@OR13] I suggest you make your suggestions on that open PR, and if accepted we merge this PR with your changes, if that working group does not accept your changes, I don't think we should deviate from how they are registering things.

The JSON-LD WG does not hold a patent on "how [to] register things [with IANA]". There are a great many **IANA Considerations** sections in many TRs, and while they are all similar, they are not identical.

My suggested changes in this PR were and are meant to make _this document_ easier for readers to comprehend. If you think my changes make things more difficult to understand, I suggest you explain how and why, especially but not only because you approved this PR before my last (relatively small!) commit which only adjusted the subsection title, which change is not what you are now disagreeing with.

**Review by dlongley — APPROVED** [review]

**Comment by TallTed** [comment]
> [@OR13] your PR here deviates from how the JSON-LD WG requested registration, and in particular what the sidebar navigation looks like.

My PR here does _not_ change how the VC WG requested registration within this document, which already "[deviated]" (by which I think you mean "[differed]") "from how the JSON-LD WG requested registration".

First thing, I say again, the JSON-LD WG has no power nor influence over how other WGs request Media Type registration. Pointing to another WG's output is only useful as an example of something; it does not indicate that other WGs should do as that example WG did.

Note that my PR here adds 3 lines of HTML and changes 2 others. That is far fewer than would be required to get from "how the JSON-LD WG requested registration" to "how the VC WG requested registration".

My PR here does indeed change what the (collapsible, non-normative, document-navigation) sidebar looks like, because it changes the subsection titles by adding details which make the content of those subsections clearer to readers who did not participate in construction of the document.

It seems you might be happier if I broke these subsections down further (which would be acceptable to me), to change —
```
  <h2>JSON Web Token Claim Registrations: `vc` and `vp`</h2>
...
  <h2>Media Type Registration: `application/vc+ld+json`</h2>
...
  <h2>Media Type Registration: `application/vp+ld+json`</h2>
...
``` 
— to —
```
  <h2>JSON Web Token Claim Registrations</h2>
    <h3>`vc`</h3>
...
    <h3>`vp`</h3>
...
  <h2>Media Type Registrations</h2>
    <h3>`application/vc+ld+json`</h3>
...
    <h3>`application/vp+ld+json`</h3>
...
```

As to "how the JSON-LD WG requested registration", I much prefer the tabular layout used by the VC WG to the sub-sub-headers and sub-sub-bodies used by the JSON-LD WG. I would not argue with treating the first column as `th` instead of `td`.

**Comment by OR13** [comment]
@TallTed those suggestions seem good, I would approve if you made them.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-07-11-vcwg-special#section1-5) on 2023-07-11

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.5. Rationalize and clarify IANA Considerations (pr vc-data-model#1163)

_See github pull request [vc-data-model#1163](https://github.com/w3c/vc-data-model/pull/1163)._




**Brent Zundel:** rationalise and clarify IANA considerations.  

**Ted Thibodeau Jr.:** This P.R intends to add additional clarity to section headings and ordering.  
… objection seems to be in relation to aesthetic breakage of sidebar.  


**Ted Thibodeau Jr.:** the duplications can be eliminated by tweaking the headings.  

**Orie Steele:** fine with changing the order and adding clarifying text. Just don't like respec sidebars with really long headers.  
… must have missed TallTed suggestion. Will take another look.  

</details>

File: index.html
Comment (TallTed) [inline index.html:5181]:
```suggestion
           <h4>`application/vp+ld+json`</h4>
```

File: index.html
Comment (TallTed) [inline index.html:5132]:
```suggestion
              <td>`application`</td>
            </tr>
            <tr>
              <td>Subtype name: </td>
              <td>`vc+ld+json`</td>
```

File: index.html
Comment (TallTed) [inline index.html:5193]:
```suggestion
              <td>`application`</td>
            </tr>
            <tr>
              <td>Subtype name: </td>
              <td>`vp+ld+json`</td>
```

File: index.html
Comment (dlongley) [inline index.html:5319]:
```suggestion
                <th>Claim Description: </th>
```

File: index.html
Comment (dlongley) [inline index.html:5480]:
```suggestion
            indicated by the presence of `ld+json` in the media type, the presentation is expected
```

**Review by dlongley — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-07-11-vcwg-special#section1-5) on 2023-07-11

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.5. Rationalize and clarify IANA Considerations (pr vc-data-model#1163)

_See github pull request [vc-data-model#1163](https://github.com/w3c/vc-data-model/pull/1163)._




**Brent Zundel:** rationalise and clarify IANA considerations.  

**Ted Thibodeau Jr.:** This P.R intends to add additional clarity to section headings and ordering.  
… objection seems to be in relation to aesthetic breakage of sidebar.  


**Ted Thibodeau Jr.:** the duplications can be eliminated by tweaking the headings.  

**Orie Steele:** fine with changing the order and adding clarifying text. Just don't like respec sidebars with really long headers.  
… must have missed TallTed suggestion. Will take another look.  

</details>

**Pull request merged**
